src/telemetryReporterWrapper.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import * as vscode from "vscode";
+import TelemetryReporter from "vscode-extension-telemetry";
+import { Constants } from "./constants";
+import { NSAT } from "./nsat";
+
+export class TelemetryReporterWrapper extends TelemetryReporter {

```

Comment:
What is the difference in purpose between this class and current telemetryClient? Can we use telemetryClient instead of adding a new telemetry class?